src/renderer/features/wallet-details/model/wallet-proxies-model.ts

```diff
@@ -1,28 +1,34 @@
 import { type ApiPromise } from '@polkadot/api';
-import { combine } from 'effector';
+import { combine, createEvent, sample } from 'effector';
 import { createGate } from 'effector-react';
 
 import { proxyService } from '@/shared/api/proxy';
 import { type Chain, type ChainId, type Wallet } from '@/shared/core';
 import { type ProxyAccount } from '@/shared/core/types/proxy';
 import { createStoreFromEffect } from '@/shared/effector';
-import { entries, keys, nonNullable, nullable } from '@/shared/lib/utils';
-import { type AnyAccount, accountService, accounts } from '@/domains/network';
+import { keys, nonNullable } from '@/shared/lib/utils';
+import { type AnyAccount, accountService, accountSync, accounts } from '@/domains/network';
 import { networkModel, networkUtils } from '@/entities/network';
-import { accountUtils } from '@/entities/wallet';
+
+export type Proxy = Omit<ProxyAccount, 'id' | 'delay'> & { deposit: string };
+type WalletProxiesByChain = Record<ChainId, Proxy[]>;
 
 const flow = createGate<{ wallet: Wallet | null }>({ defaultState: { wallet: null } });
 
 const $wallet = flow.state.map(({ wallet }) => wallet);
 
-const { $: $walletProxies } = createStoreFromEffect<
+const {
+  $: $walletProxies,
+  fx: _fetchWalletProxiesFx,
+  $pending: $walletProxiesPending,
+} = createStoreFromEffect<
```

Comment:
This is still not a subscription. We should subscribe proxies on each chain instead of `getProxiesForAccount`. I suggest to ask @johnthecat how to achieve that.
If we dont do this, there will always be bugs about "proxy list is not up-to-date after X"

Reply:
Spoke with @johnthecat and refactored it to `createSubscriptionResource` and `subscribeToChainsFx`